build-local.sh

```diff
@@ -48,7 +48,10 @@ while (( "$#" )); do
   esac
 done
 
-docker-compose -f docker-compose.build.yml down -v --remove-orphans
+docker-compose -f docker-compose.build.yml down --remove-orphans
+
+# Remove all volumes except for the go-modules volume
+docker volume ls --format='{{.Name}}' | grep optimism-integration | grep -v go-modules | xargs docker volume rm -f
```

Comment:
Can you split this up into multiple lines? It is too much for one line. Can escape newlines with the pipe as the first character on the newline for style consistency, see: https://github.com/ethereum-optimism/go-ethereum/blob/283fb9393cd0b63d495213ad8471a9f5580521eb/docker/verifier_entrypoint.sh#L22

Also can you pipe stdout/stderr to `/dev/null` so that an error message doesn't appear in the case of no volumes present?

Reply:
Sure thing and done!